src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -55,6 +62,12 @@ const HotspotDetails = ({ hotspot }: { hotspot?: Hotspot }) => {
 
   const [timelineValue, setTimelineValue] = useState(14)
 
+  const syncStatus = useMemo(() => {
+    if (!hotspot?.block) return
+
+    return getSyncStatus(hotspot.block, blockHeight)
```

Comment:
I forget what this block is, but we should be using hotspot.status.height instead. Also I think we should only show "Needs Attention" if the hotspot.status.online does not equal "online" to match explorer. The status data is the gossiped data that we use for online state and is also what controls the offline / online notifications so we need to match that.